### DISCUSSION TITLE
Same spelling but different case?

### DISCUSSION THREAD

**Discussion by ChaseOnTheWeb** [sapegin/textlint-rule-terminology#79] [Q&A]
I am working in a project where the documentation first added IDs (pluralization of ID) to its terminology dictionary, but now needs to add IDS (abbreviation for intrusion detection system) as well.

Is there a way to allow case sensitivity?

No matter what order I try to put them in it wants me to change "IDS" to "IDs" or vice versa:
```
  1:15   ✓ error  Incorrect term: “IDs”, use “IDS” instead  terminology
  1:101  ✓ error  Incorrect term: “IDs”, use “IDS” instead  terminology
```

`example.txt`:
```md
Ideally, User IDs should be randomly generated to prevent the creation of predictable or sequential IDs, which could pose a security risk.

Implement intrusion detection systems (IDS) for real-time monitoring. \
Snort IDS rule example:
`alert tcp any any -> $HOME_NET 80 (msg:"Possible SQL injection attempt"; content:"SELECT * FROM"; sid:1000001;)`
```

`.textlintrc`:
```js
{
	"rules": {
		"terminology": {
			"defaultTerms": false,
			"terms": [
				[
					"id['’]s",
					"IDs"
				],
				"IDS"
			]
		}
	}
}
```

**Comment by sapegin** [comment]
Yeah, it's not really made for this kind of cases.

I think the easiest (but still generic enough to have it) solution would be to add a list of case-sensitive exclusions. They won't enforce anything but won't report matching rules for these words either.  For your case, you'd keep the normal `IDs` rule, and add `IDS` as an exclusion. I'm not sure what would be a good name, though, we already have `skip` and `exclude`... maybe `allow`?

Fell free to make a pull request for this 👾
